client/src/app/components/Layout/Layout.jsx

```diff
@@ -16,8 +16,15 @@ import Page404 from 'common/components/Page404';
 import About from 'modules/about';
 import PrivacyPolicy from 'modules/privacy-policy';
 import Cohorts from 'modules/cohort/components/Cohorts';
+import Toolbar, { ToolbarItem } from './Toolbar';
+import { ListModeIcon, TilesModeIcon } from 'assets/images/icons';
+import { Routes } from 'common/constants/enums';
+
+export class Layout extends PureComponent {
+  state = {
+    listMode: true
```

Comment:
create ENUM for this.
listMode true is not clear 
Would be better to have
```
 viewType: ViewType.LIST 
 viewType: ViewType.TILE
```

or diplayType: DisplayType.LIST   itd
you even named one of the methods as so.
more readable for future I think